definitions/artifacts/azure-storage-account-data-lake.json

```diff
@@ -27,30 +26,18 @@
           "type": "object",
           "required": [
             "ari",
-            "primary_dfs_endpoint"
+            "primary_endpoint",
+            "secondary_endpoint"
           ],
           "properties": {
             "ari": {
               "$ref": "../types/azure-resource-id.json"
             },
-            "primary_dfs_endpoint": {
-              "$ref": "../types/azure-dfs.json"
+            "primary_endpoint": {
```

Comment:
There are a few breaking changes in here. Do we have anybody provisioning with bundles that use this currently?

Reply:
Checking w/ Chris

Reply:
@coryodaniel Chris confirmed no one has `azure-storage-account-application-assets` bundle currently deployed